sdk/storage/azure-storage-blobs/src/blob_client.cpp

```diff
@@ -27,8 +27,8 @@ namespace Azure { namespace Storage { namespace Blobs {
   {
     auto parsedConnectionString = Details::ParseConnectionString(connectionString);
     auto blobUri = std::move(parsedConnectionString.BlobServiceUri);
-    blobUri.AppendPath(containerName);
-    blobUri.AppendPath(blobName);
+    blobUri.AppendPath(Details::UrlEncodePath(containerName));
```

Comment:
Just to confirm, did you audit all the calls to `AppendPath` in the SDK?

Reply:
of course i did